tools/serverpod_cli/lib/src/generator/dart/library_generators/entities_library_generator.dart

```diff
@@ -380,6 +388,13 @@ class SerializableEntityLibraryGenerator {
                       ..url = 'package:serverpod/serverpod.dart')
                     ..name = 'transaction'
                     ..named = true),
+                  Parameter((p) => p
+                    ..type = TypeReference((b) => b
+                      ..isNullable = true
+                      ..symbol = 'Include'
```

Comment:
Should define type here as well so that users can find the correct include object to use.